cog/cog-client.cpp

```diff
@@ -63,6 +65,52 @@ struct _CogClient {
 
 G_DEFINE_TYPE_WITH_PRIVATE (CogClient, cog_client, G_TYPE_OBJECT)
 
+enum {
+  PROP_REGION=1,
+  N_PROPERTIES
+};
+
+static void
+cog_client_set_property (GObject      *object,
```

Comment:
Style nitpick: We are trying not to use the full GNOME parameter alignment in new code, because there are no formatting tools that support it (for when we finally get formatting tools working on this repo.)